_intent_specs/x11.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Start an X11 Server
+description: This intention is used to allow X11 applications to be able to launch an X server as a prerrequisite for other applications.
+action: x11
+constant: android.content.Intent.ACTION_VIEW
+uri: x11://host:port. 
+link: http://docs.theqvd.com/docs/android.html#_x_server
+extras:
```

Comment:
In the code below you are not using any extras (as in parts of the intent bundle). Are they used at all?